### PR TITLE
Added Jaxa yaml

### PR DESCRIPTION
**Pull request by AlessandroScremin**
added chla and tsm for Jaxa datasets

Please check if I commited only Jaxa or the entire Fork...not sure what's happened, but if you find folders in Capital letters maybe I commited the all the FORK...

### CODE REVIEW

File: collections/JAXA_wq_chla.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+Name: JAXA_wq_chla
+Description: "water quality Chlorophil a anomalies"
```
Comment (Kampen1988) [inline collections/JAXA_wq_chla.yaml:2]: Suggestion: change *Chlorophil* to Chlorophyll for consistency

File: collections/JAXA_wq_chla.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+AdditionalInfoExternal:
+    Title: Additional info
+    Path: jaxa_wq_chla/README.MD
+Resolution:
+GeographicalCoverage: 12.0, 33.85, 140.25, 45.8
+TemporalAvailability: 2018-01-06 - 2021-04-17
+UpdateFrequency: weekly
+BandInformation:chla
```
Comment (Kampen1988) [inline collections/JAXA_wq_chla.yaml:10]: Requires space after the colon to separate key value pairs.

File: collections/JAXA_wq_chla.yaml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+Contact: "[JAXA]"
+Provider:"[JAXA]"
+ManagedBy:"[ESA](http://www.esa.int/)"
```
Comment (Kampen1988) [inline collections/JAXA_wq_chla.yaml:13]: Requires space after the colon to separate key value pairs.

File: collections/JAXA_wq_chla.yaml [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+License: |
+ Full, open and free access
+Resources:
+  - Group: Sentinel Hub Resources
+  - EndPoint: shservices.mundiwebservices.com
+   Type: byoc-dc461d36-d855-4021-93e6-686f08d48f3d
+   CollectionId: dc461d36-d855-4021-93e6-686f08d48f3d
+
+RegistryEntryAdded: "2021-05-26"
+RegistryEntryLastModified: "2021-05-26"
```
Comment (Kampen1988) [inline collections/JAXA_wq_chla.yaml:28]: Consider adding an empty line at the end of the file.

**Review by Kampen1988 — CHANGES_REQUESTED** [review]
Commented sections that require minor editing

File: collections/jaxa_wq_tsm/README.MD [hunk 5]
```diff
@@ -0,0 +1,18 @@
+# JAXA_wq_tsm
+
+## Short description
+Chlorophyll-a concentration is an indicator of algae abundance which fluctuates naturally over space and time, as a result of combined atmospheric and oceanic effects (e.g., marine currents and upwelling). In coastal areas, strongly influenced by river inputs and human activities, high Chl concentration can result from the discharge of urban sewage, industrial runoffs, and fertilizers from agriculture activities over watersheds. In particular, nutrient inputs of anthropogenic origin affect the natural amount of phytoplankton in marine and inland waters, representing a continuous threat to biodiversity and leading to undesirable modifications of phytoplankton concentration (i.e., eutrophication).
```
[the remark this answers is not in the document]
Comment (AlessandroScremin) [reply collections/jaxa_wq_tsm/README.MD]: rearranged and added image in description

File: collections/jaxa_wq_chla/README.MD [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+
+Total Suspended Matter (TSM) i.e, the concentration of organic and inorganic materials suspended in the water, is another proxy for water quality as different contaminants, including nutrients, trace metals, semi-volatile organic compounds, and numerous pesticides, can aggregate to these solids and brought in suspension. This can alter the state of the aquatic ecosystem and the use of freshwater resources. For instance, excessive suspended material might condition primary productivity. TSM concentration can be very high near the coasts due to the resuspension of terrestrial or submarine particulate matter by tides, waves, and currents.
```
[the remark this answers is not in the document]
Comment (AlessandroScremin) [reply collections/jaxa_wq_chla/README.MD]: rearranged and added image in description

File: collections/JAXA_wq_chla.yaml (hunk 4 above)
Comment (AlessandroScremin) [reply collections/JAXA_wq_chla.yaml]: done

File: collections/JAXA_wq_chla.yaml (hunk 1 above)
Comment (AlessandroScremin) [reply collections/JAXA_wq_chla.yaml]: done

File: collections/JAXA_wq_chla.yaml (hunk 2 above)
Comment (AlessandroScremin) [reply collections/JAXA_wq_chla.yaml]: Done

File: collections/JAXA_wq_chla.yaml [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+AdditionalInfoExternal:
+    Title: Additional info
+    Path: jaxa_wq_chla/README.MD
+Resolution:
+GeographicalCoverage: 12.0, 33.85, 140.25, 45.8
+TemporalAvailability: 2018-01-06 - 2021-04-17
+UpdateFrequency: weekly
+BandInformation:chla
+Contact: "[JAXA]"
+Provider:"[JAXA]"
```
[the remark this answers is not in the document]
Comment (AlessandroScremin) [reply collections/JAXA_wq_chla.yaml]: Done

File: collections/JAXA_wq_chla.yaml (hunk 3 above)
Comment (AlessandroScremin) [reply collections/JAXA_wq_chla.yaml]: Done

File: collections/JAXA_wq_tsm.yaml [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+AdditionalInfoExternal:
+    Title: Additional info
+    Path: jaxa_wq_tsm/README.MD
+Resolution:
+GeographicalCoverage: 12.0, 33.85, 140.25, 45.8
+TemporalAvailability: 2018-01-06 - 2021-04-17
+UpdateFrequency: weekly
+BandInformation: tsm
+Contact: "[JAXA]"
+Provider:"[JAXA]"
```
[the remark this answers is not in the document]
Comment (AlessandroScremin) [reply collections/JAXA_wq_tsm.yaml]: Done

File: collections/JAXA_wq_tsm.yaml [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
+ManagedBy:"[ESA](http://www.esa.int/)"
```
[the remark this answers is not in the document]
Comment (AlessandroScremin) [reply collections/JAXA_wq_tsm.yaml]: Done

**Review by Kampen1988 — APPROVED** [review]

File: collections/JAXA_wq_tsm.yaml [hunk 10]
```diff
@@ -0,0 +1,28 @@
...
+    - sentinel hub
+    - raster
+    - dashboard hackathon
+License: |
+ Full, open and free access
+Resources:
+  - Group: Sentinel Hub Resources
+  - EndPoint: shservices.mundiwebservices.com
+   Type: byoc-4f5f67f1-5715-4f2b-8c98-ae57948ee2f5
+   CollectionId: 4f5f67f1-5715-4f2b-8c98-ae57948ee2f5
```
Comment (Kampen1988) [inline collections/JAXA_wq_tsm.yaml:25]: Please see indentation error above.

File: collections/JAXA_wq_tsm.yaml [hunk 11]
```diff
@@ -0,0 +1,28 @@
...
+Tags:
+    - open data
+    - sentinel hub
+    - raster
+    - dashboard hackathon
+License: |
+ Full, open and free access
+Resources:
+  - Group: Sentinel Hub Resources
+  - EndPoint: shservices.mundiwebservices.com
```
Comment (Kampen1988) [inline collections/JAXA_wq_tsm.yaml:23]: Please see comment on wrongly placed bullet point above.

**Review by Kampen1988 — CHANGES_REQUESTED** [review]
There are some minor errors to be fixed. The build crashed on my local machine due to the commented indentation error.

**Comment by Kampen1988** [comment]
Hi @AlessandroScremin ,

Which software are you using for creating the `.yaml` files? VSCode for example is having a handy [yaml language support package](https://marketplace.visualstudio.com/items?itemName=redhat.vscode-yaml) that checks your files automatically and highlights errors.

Cheers, Max

**Review by Kampen1988 — APPROVED** [review]

**Pull request merged**